Formula/grpcui.rb

```diff
@@ -0,0 +1,22 @@
+class Grpcui < Formula
+  desc "An interactive web UI for gRPC, along the lines of postman"
+  homepage "https://github.com/fullstorydev/grpcui"
+  url "https://github.com/fullstorydev/grpcui/archive/v1.0.0.tar.gz"
+  sha256 "2169f72d23a5e4caceee3b863220bf6a4b4a15f7d88d5bb0de909ca4299a92df"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
```

Comment:
Please try and build without the deprecated GOPATH requirements.

Reply:
I see you fixed this for me, as well as the style issue in the description. Thanks!